rand_distr/src/dirichlet.rs

```diff
@@ -107,9 +264,7 @@ where
         if size < 2 {
             return Err(Error::SizeTooSmall);
         }
-        Ok(Dirichlet {
-            alpha: vec![alpha; size].into_boxed_slice(),
-        })
+        Ok(Dirichlet::new(vec![alpha; size].as_slice()).unwrap())
```

Comment:
Can we propagate the error instead of unwrapping?

Reply:
Gone -- `new_with_size()` was removed.